### ISSUE TITLE
`cupy.clip` cannot be fused due to unusual argument (un)packing behaviour in `cupy._core.fusion._call_ufunc()`

### ISSUE THREAD

**Issue by berryboy2012** [cupy/cupy#7098]
### Description

While `cupy.clip` is a `ufunc` and should work inside a fused kernel. In reality it doesn't. 
After some debugging, it seems that some code is behaving strangely: 
https://github.com/cupy/cupy/blob/f2833e1d9b640a6b52209506e8bcbf87cff451fe/cupy/_core/fusion.pyx#L960-L961 
To me, it should be something like this:
```py
def _call_ufunc(fusion_op, *args, **kwargs):
    return _thread_local.history.call_ufunc(fusion_op, *args, **kwargs)
```
After I changed this bit, `cupy.clip` can now work inside a fused kernel with some other caveats (setting `a_min` or `a_max` to `None` still doesn't work, but I believe that's a different story).

### To Reproduce

```py
# Write the code here
import cupy as cp
@cp.fuse
def clip_it(arr):
    return cp.clip(arr, a_min=0, a_max=1e30)
print(clip_it(cp.linspace(0,10,50)))
```

### Installation

Built from GitHub source

### Environment

```
# Paste the output here
OS                           : Linux-5.19.6-100.fc35.x86_64-x86_64-with-glibc2.34
Python Version               : 3.10.7
CuPy Version                 : 12.0.0a2
CuPy Platform                : NVIDIA CUDA
NumPy Version                : 1.23.3
SciPy Version                : None
Cython Build Version         : 0.29.32
Cython Runtime Version       : 0.29.32
CUDA Root                    : /usr/local/cuda
nvcc PATH                    : ccache nvcc
CUDA Build Version           : 11070
CUDA Driver Version          : 11070
CUDA Runtime Version         : 11070
cuBLAS Version               : (available)
cuFFT Version                : 10702
cuRAND Version               : 10210
cuSOLVER Version             : (11, 4, 0)
cuSPARSE Version             : (available)
NVRTC Version                : (11, 7)
Thrust Version               : 101500
CUB Build Version            : 101500
Jitify Build Version         : 4a37de0
cuDNN Build Version          : 8401
cuDNN Version                : 8401
NCCL Build Version           : None
NCCL Runtime Version         : None
cuTENSOR Version             : None
cuSPARSELt Build Version     : None
Device 0 Name                : NVIDIA GeForce RTX 3090
Device 0 Compute Capability  : 86
Device 0 PCI Bus ID          : 0000:01:00.0
```

**Comment by asi1024** [comment]
@berryboy2012 I tried your reproducer but worked as we expected. Could you paste here your outputs or error messages?

**Comment by berryboy2012** [comment]
> @berryboy2012 I tried your reproducer but worked as we expected. Could you paste here your outputs or error messages?

My bad, should be something as follows:
```python
# Write the code here
import cupy as cp
@cp.fuse
def clip_it(arr):
    arr = arr / cp.sum(arr)
    return cp.clip(arr, a_min=0, a_max=1e30)
print(clip_it(cp.linspace(0,10,50)))
```
Without the fix, the output will be something like this:
```
Traceback (most recent call last):
  File "cupy/_core/fusion.pyx", line 903, in cupy._core.fusion.Fusion.__call__
  File "cupy/_core/fusion.pyx", line 744, in cupy._core.fusion._FusionHistory.get_fusion
  File "/home/berry/.config/JetBrains/PyCharm2022.2/scratches/scratch.py", line 4, in clip_it
    arr = arr / cp.sum(arr)
  File "cupy/_core/fusion.pyx", line 227, in cupy._core.fusion._FusionVarScalar.__truediv__
  File "cupy/_core/_kernel.pyx", line 1177, in cupy._core._kernel.ufunc.__call__
  File "cupy/_core/_fusion_thread_local.pyx", line 35, in cupy._core._fusion_thread_local.call_ufunc
  File "cupy/_core/fusion.pyx", line 961, in cupy._core.fusion._call_ufunc
  File "cupy/_core/fusion.pyx", line 581, in cupy._core.fusion._FusionHistory.call_ufunc
  File "cupy/_core/fusion.pyx", line 521, in cupy._core.fusion._FusionHistory._get_fusion_var
Exception: Shape mismatch

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/berry/.config/JetBrains/PyCharm2022.2/scratches/scratch.py", line 6, in <module>
    print(clip_it(cp.linspace(0,10,50)))
  File "cupy/_core/fusion.pyx", line 909, in cupy._core.fusion.Fusion.__call__
  File "cupy/_core/new_fusion.pyx", line 158, in cupy._core.new_fusion.Fusion.__call__
  File "cupy/_core/new_fusion.pyx", line 29, in cupy._core.new_fusion._get_fused_kernel
  File "cupy/_core/_fusion_trace.pyx", line 595, in cupy._core._fusion_trace.trace
  File "cupy/_core/_fusion_trace.pyx", line 537, in cupy._core._fusion_trace.TraceImpl.trace
  File "/home/berry/.config/JetBrains/PyCharm2022.2/scratches/scratch.py", line 5, in clip_it
    return cp.clip(arr, a_min=0, a_max=1e30)
  File "/home/berry/cupy/cupy/_math/misc.py", line 172, in clip
    return fusion._call_ufunc(_math.clip,
  File "cupy/_core/fusion.pyx", line 961, in cupy._core.fusion._call_ufunc
  File "cupy/_core/_fusion_trace.pyx", line 290, in cupy._core._fusion_trace.TraceImpl.call_ufunc
  File "cupy/_core/_fusion_trace.pyx", line 267, in cupy._core._fusion_trace.TraceImpl._unwrap_interface
TypeError: <class 'tuple'> type is not supported

```

**Comment by asi1024** [comment]
@berryboy2012 Fixed in #7113. Thanks for your reporting!
